### PR TITLE
Tables updated

### PR DESCRIPTION
**Pull request by rs16princeton**
This pull request has the initial Styles for Tables

### CODE REVIEW

File: src/scss/_table.scss [hunk 1]
```diff
@@ -1,40 +1,101 @@
...
+  color: jazz-theme(table-row-color);
+  font-weight: normal;
+}
+
+.jazz-table tr:hover {
+  background: jazz-theme(table-row-hover-bgcolor);
+  color: jazz-theme(table-row-hover-color);
+}
+
+.jazz-table__bordered td {
```
Comment (mzarillopu) [inline src/scss/_table.scss:49]: We have been using the BEM (Block, Element, Modifier) naming convention for classes (http://getbem.com/naming/).  Following this convention, the naming should be "jazz-table--bordered".

File: src/scss/_table.scss [hunk 2]
```diff
@@ -1,40 +1,101 @@
...
+}
+
+.jazz-table__bordered th {
+  border-top: 1px solid jazz-theme(table-border-color);
+  border-left: 1px solid jazz-theme(table-border-color);
+  border-right: 1px solid jazz-theme(table-border-color);
+  border-bottom: 3px solid jazz-theme(table-th-border-color);
+}
+
+.jazz-table__open th,
```
Comment (mzarillopu) [inline src/scss/_table.scss:60]: Following BEM, the name for this class should be "jazz-table--open"

File: src/scss/_table.scss [hunk 3]
```diff
@@ -1,40 +1,101 @@
...
+.jazz-table__open td {
+  padding-top: px-to-rem(16);
+  padding-bottom: px-to-rem(16);
 }
 
-.jazz-table__borderless tr:nth-child(even) {
+.jazz-table.jazz-table__striped tr:nth-child(even):not(.jazz-table__subtotal):not(.jazz-table__grand-total):not(:hover) {
```
Comment (mzarillopu) [inline src/scss/_table.scss:66]: Following BEM, the name of the class should be "jazz-table--striped".

File: src/scss/_table.scss [hunk 4]
```diff
@@ -1,40 +1,101 @@
...
+  color: jazz-theme(table-subtotal-color);
+}
+
+.jazz-table tr.jazz-table__grand-total {
+  font-weight: bold;
+  background-color: jazz-theme(table-grand-total-bgcolor);
+  color: jazz-theme(table-grand-total-color);
+}
+
+.jazz-table__responsive {
```
Comment (mzarillopu) [inline src/scss/_table.scss:89]: Following BEM, the name of the class should be "jazz-table--responsive"

File: src/scss/_table.scss [hunk 5]
```diff
@@ -1,40 +1,101 @@
...
+  display: block;
+  width: 100%;
+  overflow-x: auto;
+  -webkit-overflow-scrolling: touch;
+}
+
+.jazz-table__no-data {
```
Comment (mzarillopu) [inline src/scss/_table.scss:96]: The naming for this class appears to be correct because "no-data" is a sub-element of the table.

**Review by mzarillopu — CHANGES_REQUESTED** [review]

File: src/scss/_table.scss (hunk 5 above)
Comment (rs16princeton) [reply src/scss/_table.scss:96]: My bad, I just kept the way the styles were named before. Should I change for the other styles too?

Comment (rs16princeton) [reply src/scss/_table.scss:96]: Nevermind I saw the other comments thanks!

File: src/scss/_table.scss [hunk 6]
```diff
@@ -1,40 +1,101 @@
...
+  padding-top: px-to-rem(16);
+  padding-bottom: px-to-rem(16);
+}
+
+.jazz-table.jazz-table--striped tr:nth-child(even):not(.jazz-table--subtotal):not(.jazz-table--grand-total):not(:hover) {
   background-color: jazz-theme(table-row-even-odd-bgcolor);
   color: jazz-theme(table-row-even-odd-color);
 }
+
+.jazz-table tr.jazz-table--subtitle {
```
Comment (mzarillopu) [inline src/scss/_table.scss:71]: This name should be "jazz-table__subtitle".  Referring back to the "BEM" naming scheme, the "subtitle" would be a sub-element of table.  Elements are separated with underscores.  Modifiers (such as "disabled" or "bordered") are separated by dashes.

File: src/scss/_table.scss [hunk 7]
```diff
@@ -1,40 +1,101 @@
...
+  font-weight: bold;
+  background-color: jazz-theme(table-subtitle-bgcolor);
+  color: jazz-theme(table-subtitle-color);
+}
+
+.jazz-table tr.jazz-table--subtotal {
```
Comment (mzarillopu) [inline src/scss/_table.scss:77]: Name should be "jazz-table__subtotal".

File: src/scss/_table.scss [hunk 8]
```diff
@@ -1,40 +1,101 @@
...
+  font-weight: bold;
+  background-color: jazz-theme(table-subtotal-bgcolor);
+  color: jazz-theme(table-subtotal-color);
+}
+
+.jazz-table tr.jazz-table--grand-total {
```
Comment (mzarillopu) [inline src/scss/_table.scss:83]: Name should be "jazz-table__grand-total".

**Review by mzarillopu — CHANGES_REQUESTED** [review]

**Review by mzarillopu — APPROVED** [review]

**Pull request merged**